src/pages/search/searchView.js

```diff
@@ -1,57 +1,63 @@
-/* eslint-disable */
 import React from 'react';
 import {
-  TextField, CircularProgress, withStyles, Box, ListItem, Divider, Chip, Tabs, Tab
+  TextField, CircularProgress, withStyles, Box, Tab, Popper,
 } from '@material-ui/core';
-import SearchIcon from '@material-ui/icons/Search';
-import Autocomplete from '@material-ui/lab/Autocomplete';
-import TabContext from '@material-ui/lab/TabContext';
-import TabList from '@material-ui/lab/TabList';
-import TabPanel from '@material-ui/lab/TabPanel';
-
+import {
+  Autocomplete, TabContext, TabList, TabPanel,
+} from '@material-ui/lab';
+import { Search as SearchIcon, FilterList as FilterIcon } from '@material-ui/icons';
 import { getSearch, getSearchPageResults } from '../dashboardTab/store/dashboardReducer';
 import Pagination from './components/pagination';
 import Subsection from './components/searchCard';
 
-
 function searchComponent({ classes, searchparam = '' }) {
   const [tab, setTab] = React.useState('2');
 
   const handleChange = (event, newValue) => {
     setTab(newValue);
   };
 
-  const [open, setOpen] = React.useState(false);
+  const [open] = React.useState(false);
   const [inputValue, setInputValue] = React.useState('');
   const [options, setOptions] = React.useState([]);
   const [searchResults, setSearchResults] = React.useState([]);
   const loading = open;
-  const [value, setValue] = React.useState([]);
-
-  React.useEffect(() => {
-    if (!open) {
-      setOptions([]);
-    }
-    getAutoCompleteRes(searchparam)
-    onChange(searchparam);
-  }, [open]);
+  const [value] = React.useState([]);
 
   async function onChange(newValue = []) {
     const searchResp = await getSearchPageResults(newValue);
     setSearchResults(searchResp);
   }
 
+  const CustomPopper = (props) => <Popper {...props} className={classes.root} placement="bottom" />;
```

Comment:
Codacy has a fix for the issue: [Function component is not a function expression](https://app.codacy.com/gh/CBIIT/bento-frontend/pullRequest?prid=8533938)
```suggestion
  var CustomPopper = function(props) {
```